src/trrntzip.c

```diff
@@ -984,7 +997,7 @@ int main(int argc, char **argv) {
 
   if (argc < 2 || iOptionsFound == (argc - 1)) {
     fprintf(stderr, "trrntzip: missing path\n");
-    fprintf(stderr, "Usage: trrntzip [-dfghqsv] [PATH/ZIP FILE]\n");
+    fprintf(stderr, "Usage: trrntzip [-defghlqsv] [PATH/ZIP FILE]\n");
```

Comment:
I'll change the usage strings.

We have to handle `-e` withe empty `FOO` somehow. We could treat it as an error, use the defaults, or disable the log. I chose the last option because that's most useful. (Even more with `-l`: to disable process logs and keep the error log you can pass `-l -eerror.log`.)

Reply:
Sure, that's fine.